<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtPrediction</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="twResults">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Params</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Models</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>List</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Status</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QListWidget" name="lwModelsList"/>
           </item>
           <item row="1" column="1">
            <widget class="QListWidget" name="lwModelDescription"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QTextEdit" name="textEditDescription">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Prediction</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEditTextToPredict">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="predictButton">
             <property name="text">
              <string>Predict</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Result:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lPredictResult">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>unknown</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QListWidget" name="lwPredictResult"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Add to model</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Label</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbAddToModel">
               <property name="editable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbAddToModel">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Add to model</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Text</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>2</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="teHTTPReplies"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Test</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0" rowspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Results</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPlainTextEdit" name="teTestResults"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbTestClearResults">
             <property name="text">
              <string>Clear results</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Notice! Please select &amp;quot;moscow_weather&amp;quot; in the models list (page &amp;quot;Params&amp;quot;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>test weather</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>from</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="sbTestFrom">
             <property name="maximum">
              <number>360</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>to</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sbTestTo">
             <property name="maximum">
              <number>360</number>
             </property>
             <property name="value">
              <number>360</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>season</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="cbTestSeason">
             <item>
              <property name="text">
               <string>Winter</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Spring</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Summer</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Fall</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>step</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="sbTestStep">
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QPushButton" name="pbTestStart">
             <property name="text">
              <string>Start/Stop</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QProgressBar" name="pbTestWeather">
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>now</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="sbTestNow">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="minimum">
              <number>-100</number>
             </property>
             <property name="maximum">
              <number>600</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="separator"/>
    <addaction name="actionNewModel"/>
    <addaction name="actionDeleteModel"/>
    <addaction name="actionUpdateModelsList"/>
    <addaction name="actionUpdateModelInfo"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewModel"/>
   <addaction name="actionDeleteModel"/>
   <addaction name="actionUpdateModelsList"/>
  </widget>
  <action name="actionAPI_Key">
   <property name="text">
    <string>API Key</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login...</string>
   </property>
  </action>
  <action name="actionNewModel">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New model...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDeleteModel">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/edit-delete.png</normaloff>:/images/images/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete model...</string>
   </property>
  </action>
  <action name="actionUpdateModelsList">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/view-refresh.png</normaloff>:/images/images/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Update models list</string>
   </property>
  </action>
  <action name="actionUpdateModelInfo">
   <property name="text">
    <string>Update model info</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
